### PR TITLE
data:Vikkybliz

### PR DESCRIPTION
**Pull request by Vikkybliz**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Opened my personal linkfree account

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4519"></a>

### CODE REVIEW

File: data/Vikkybliz.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		},
+		{
+			"name": "Twitter: Mostly Programming and Technology tweets",
+			"url": "https://twitter.com/vikkybliz",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Want to learn FLutter? Go here!",
+			"url": "https://www.youtube.com/channel/UCe6ciVTGIiDRd7dbIdzAXig/",
+			"icon": "SiHashnode"
```
Comment (priyankarpal) [inline data/Vikkybliz.json:37]: You added a wrong Icon. Change it to a YouTube icon.
``` suggestion
			"icon": "SiYoutube"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**